recipes/ignition-cmake/all/test_package/conanfile.py

```diff
@@ -5,7 +5,7 @@
 
 class TestPackageConan(ConanFile):
     settings = "os", "compiler", "build_type", "arch"
-    generators = "cmake", "cmake_find_package_multi"
+    generators = "cmake", "cmake_find_package"
```

Comment:
why need to change it now?

Reply:
I was doing some testing and forgot to revert it back. Reverted it now. thanks for pointing out